### PR TITLE
✨ Source S3: v4 rollout - take 3

### PR DESCRIPTION
**Pull request by maxi297**
Duplicate of https://github.com/airbytehq/airbyte/pull/30147 to remove source-hubspot because of auto-formatiing

### CODE REVIEW

**Comment by maxi297** [comment]
Breaking change document: https://docs.google.com/document/d/1WSNobLTN0aXE5ZMfCj6WHRzqMFnmVqJBMFYXljBj5pI/edit

**Review by clnoll — APPROVED** [review]
Just a few small questions/suggestions but other than that LGTM @maxi297!

File: airbyte-integrations/connectors/source-s3/source_s3/v4/config.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def documentation_url(cls) -> AnyUrl:
         description="In order to access private Buckets stored on AWS S3, this connector requires credentials with the proper "
         "permissions. If accessing publicly available data, this field is not necessary.",
         airbyte_secret=True,
-        order=1,
```
[the remark this answers is not in the document]
Comment (maxi297) [reply airbyte-integrations/connectors/source-s3/source_s3/v4/config.py:29]: I don't but CATs does ; it fails on duplicated order

[the remark this answers is not in the document]
Comment (clnoll) [reply airbyte-integrations/connectors/source-s3/source_s3/v4/config.py:29]: Ah got it, I see an `order=1` in the file-based spec.

**Pull request merged**